core/src/main/scala/chisel3/BitsImpl.scala

```diff
@@ -3,8 +3,9 @@
 package chisel3
 
 import chisel3.experimental.{requireIsHardware, SourceInfo}
-import chisel3.internal.{_resizeToWidth, throwException, BaseModule}
+import chisel3.internal.{throwException, BaseModule}
 import chisel3.internal.Builder.pushOp
+import chisel3.internal._
```

Comment:
```suggestion
```
Don't add a wildcard import unless we really need to (and this PR shouldn't need it).